projects/plugins/jetpack/extensions/blocks/contact-form/util/form-styles.js

```diff
@@ -0,0 +1,86 @@
+function getBackgroundColor( backgroundColorNode ) {
```

Comment:
Because this file is not compiled and goes into the global scope as is, I think it might be best to give it a slightly more specific name or make it a property on a global `window.jetpackForms` or something. Or just make it private.